### PR TITLE
Allow resizing of CSF contained levels

### PR DESCRIPTION
**Pull request by balderdash**
The HeightResizer actually works now (it used to be fixed in place, since both the minHeight and maxHeight were set to the height of the contained level), with scrolling when you make it smaller.

### CODE REVIEW

File: apps/src/templates/instructions/TopInstructionsCSF.jsx [hunk 1]
```diff
@@ -148,9 +149,17 @@ const styles = {
...
+    backgroundColor: color.background_gray,
+    overflowY: 'scroll',
+  },
   containedLevel: {
-    padding: CONTAINED_LEVEL_PADDING,
-    backgroundColor: '#f2f2f2',
+    paddingTop: CONTAINED_LEVEL_PADDING,
+    paddingLeft: CONTAINED_LEVEL_PADDING,
+    paddingRight: CONTAINED_LEVEL_PADDING,
+  },
```
Comment (Hamms) [inline apps/src/templates/instructions/TopInstructionsCSF.jsx]: nit: might be more extensible to put these into a `containedLevelStyles` object, a la the `craftStyles`

File: dashboard/app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1791,6 +1791,10 @@ a.download-video {
   box-sizing: border-box;
 }
 
+.contained-level .free-response .response {
+  resize: vertical;
+}
```
Comment (Hamms) [inline dashboard/app/assets/stylesheets/application.scss:1796]: why is this here and not in an apps stylesheet?

File: apps/src/templates/instructions/TopInstructionsCSF.jsx [hunk 3]
```diff
@@ -509,30 +526,37 @@ var TopInstructions = React.createClass({
...
+          >
+            <div style={styles.containedLevel} className="contained-level">
+              <ContainedLevel ref="containedLevel" />
+            </div>
           </div>
           {!this.props.collapsed && !this.props.isEmbedView &&
             <HeightResizer
               position={this.props.height}
               onResize={this.handleHeightResize}
+              style={styles.heightResizer}
```
Comment (Hamms) [inline apps/src/templates/instructions/TopInstructionsCSF.jsx]: HeightResizer doesn't have a `style` property; what is this intended to do?

File: apps/src/templates/instructions/TopInstructionsCSF.jsx (hunk 1 above)
Comment (balderdash) [reply apps/src/templates/instructions/TopInstructionsCSF.jsx]: Done.

File: apps/src/templates/instructions/TopInstructionsCSF.jsx (hunk 3 above)
Comment (balderdash) [reply apps/src/templates/instructions/TopInstructionsCSF.jsx]: I added support for it in this PR. Is it bad practice to add a style property to react components?

File: dashboard/app/assets/stylesheets/application.scss (hunk 2 above)
Comment (balderdash) [reply dashboard/app/assets/stylesheets/application.scss:1796]: Mainly because this is where the rest of the free response level styling is.

File: apps/src/templates/instructions/TopInstructionsCSF.jsx (hunk 3 above)
Comment (Hamms) [reply apps/src/templates/instructions/TopInstructionsCSF.jsx]: Oh! Wow, I totally missed that. Nope, what you did is perfect.

File: dashboard/app/assets/stylesheets/application.scss (hunk 2 above)
Comment (Hamms) [reply dashboard/app/assets/stylesheets/application.scss:1796]: Man, our CSS needs some normalization. This is a fine place, in that case.
